docs/requirements.txt

```diff
@@ -1,25 +0,0 @@
-alabaster==0.7.7
-argh==0.26.1
```

Comment:
Yea, as discussed in #114 this was an attempt at making syntax highlighting work, and it didn't work, so removing again.